### PR TITLE
DOCS-2107: Update robot.go

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: robot/robot.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ import (
 // GetCloudMetadata example:
 //
 //	metadata := machine.GetCloudMetadata()
```
Comment (npentrel) [inline robot/robot.go:69]:
```suggestion
//	metadata, err := machine.CloudMetadata(ctx.Background())
```

File: robot/robot.go [hunk 2]
```diff
@@ -80,6 +81,12 @@ import (
 //
 //	// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
 //	err := machine.StopAll(ctx)
+//
+// Shutdown example:
+//
+// 	// Shutdown down the robot.
```
Comment (npentrel) [inline robot/robot.go:87]:
```suggestion
// 	// Shut down the robot.
```

File: robot/robot.go [hunk 3]
```diff
@@ -80,6 +81,12 @@ import (
 //
 //	// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
 //	err := machine.StopAll(ctx)
+//
+// Shutdown example:
+//
+// 	// Shut down the robot.
+//	err := machine.Shutdown(ctx)
```
Comment (sguequierre) [inline robot/robot.go:88]: Would this and above not also have to have ctx declared?

Comment (npentrel) [reply robot/robot.go:88]: actually I don't think it needs this

Comment (npentrel) [inline robot/robot.go:88]:
```suggestion
//	err := machine.Shutdown()
```

File: robot/robot.go [hunk 4]
```diff
@@ -66,7 +66,8 @@ import (
 //
 // GetCloudMetadata example:
 //
-//	metadata := machine.GetCloudMetadata()
+//	metadata, err := machine.CloudMetadata(ctx.Background())
```
Comment (npentrel) [inline robot/robot.go:69]:
```suggestion
//	metadata, err := machine.CloudMetadata()
```

**Review by njooma — APPROVED** [review]

File: robot/robot.go [hunk 5]
```diff
@@ -80,6 +81,12 @@ import (
 //
 //	// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
 //	err := machine.StopAll(ctx)
+//
+// Shutdown example:
+//
+// 	// Shut down the robot.
+//	err := machine.Shutdown()
+
```
Comment (andf-viam) [inline robot/robot.go:89]:
```suggestion
//
```

File: robot/robot.go [hunk 6]
```diff
@@ -80,6 +81,12 @@ import (
 //
 //	// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
 //	err := machine.StopAll(ctx)
+//
+// Shutdown example:
+//
+// 	// Shut down the robot.
```
Comment (andf-viam) [inline robot/robot.go:87]:
```suggestion
//	// Shut down the robot.
```

**Pull request merged**